docs/content/Configuration/Environment-Variables-Reference.md

```diff
@@ -125,36 +126,48 @@ databases [in this guide][link-connecting-to-db].
+| `CUBESTORE_SERVER_NAME`         | The full name and port number of the Cube Store server. Must be unique for each instance in cluster mode. Defaults to `localhost`       | A valid address/port pair                                   |
+| `CUBESTORE_WAL_SPLIT_THRESHOLD` | The maximum number of rows to keep in a single chunk of data right after insertion. Defaults to `262144`                                | A valid number                                              |
+| `CUBESTORE_WORKER_PORT`         | The port for Cube Store workers to listen to connections on. When set, the node will start as a **worker** in the cluster               | A valid port number                                         |
+| `CUBESTORE_WORKERS`             | A comma-separated list of address/port pairs; for example `worker-1:3123,localhost:3124,123.124.125.128:3123`                           | A comma-separated list of address/port pairs                |
+
+### Cloud Storage
+
+| Environment variable                                | Description                                                                                | Possible Values                                                               |
+| --------------------------------------------------- | ------------------------------------------------------------------------------------------ | ----------------------------------------------------------------------------- |
+| `CUBESTORE_AWS_ACCESS_KEY_ID`                       | The Access Key ID for AWS. Required when using AWS S3                                      | [A valid AWS Access Key ID][link-aws-creds]                                   |
+| `CUBESTORE_AWS_SECRET_ACCESS_KEY`                   | The Secret Access Key for AWS. Required when using AWS S3                                  | [A valid AWS Secret Access Key][link-aws-creds]                               |
+| `CUBESTORE_S3_BUCKET`                               | The name of a bucket in AWS S3                                                             | -                                                                             |
+| `CUBESTORE_S3_REGION`                               | The region of a bucket in AWS S3                                                           | -                                                                             |
+| `CUBESTORE_S3_SUB_PATH`                             | The path in a AWS S3 bucket to store pre-aggregations. Optional                            | -                                                                             |
+| `CUBESTORE_GCP_SERVICE_ACCOUNT`                     | The path to a JSON key file for Google Cloud. Required when using Google Cloud Storage     | [The path to a valid Google Cloud JSON key file][link-gcp-creds-json]         |
```

Comment:
We could also mention that it's the same as `SERVICE_ACCOUNT` for configuring Google Cloud SDK.
Folks should be familiar with it.